document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -52,8 +53,12 @@ public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequ
 
         // when the Accounts migration has been completed to Company Accounts, this code can be removed
         if (isAccounts(resourceLink)) {
-            return accountsHandler.getAccountsData(resourceLink);
-         }
+            try {
+                return accountsHandler.getAccountsData(resourceLink);
+            } catch (HandlerException e) {
+                LOG.error(e);
```

Comment:
Should this log be more detailed to aid support?

Reply:
Check the description of the PR:
>NOTE: This PR does not cover the fine tuned Exception handling or logging. It simply puts in place a skeleton in which we can build on afterwards.

;)

Reply:
My mistake!